email-reader/src/main/java/no/sikt/nva/email/reader/handler/VerifyScopusEmailReceivedHandler.java

```diff
@@ -0,0 +1,110 @@
+package no.sikt.nva.email.reader.handler;
+
+import static nva.commons.core.attempt.Try.attempt;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.ScheduledEvent;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import no.sikt.nva.email.reader.mapper.messagebodyreader.EmailParser;
+import no.sikt.nva.email.reader.mapper.messagebodyreader.ScopusEmailValidator;
+import no.sikt.nva.email.reader.model.exception.NoScopusEmailsReceived;
+import no.unit.nva.s3.S3Driver;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.StringUtils;
+import nva.commons.core.attempt.Failure;
+import nva.commons.core.paths.UnixPath;
+import org.apache.james.mime4j.dom.Message;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.ListObjectsV2Request;
+import software.amazon.awssdk.services.s3.model.S3Object;
+
+public class VerifyScopusEmailReceivedHandler
+    implements RequestHandler<ScheduledEvent, Void> {
+    private static final String NON_SCOPUS_EMAIL_FOUND = "NON SCOPUS EMAIL FOUND";
+    private static final Logger logger = LoggerFactory.getLogger(VerifyScopusEmailReceivedHandler.class);
+
+    //we expect to receive one email once a week; and we have object expiration set to 20 days.
+    private static final Integer GENEROUS_LIMIT_OF_EXPECTED_KEYS_IN_BUCKET = 10;
+    private final S3Client s3Client;
+    private final String bucketName;
+
+    @JacocoGenerated
+    public VerifyScopusEmailReceivedHandler() {
+        this(S3Client.create(),
+             new Environment().readEnv("SCOPUS_EMAIL_BUCKET_NAME"));
+    }
+
+    public VerifyScopusEmailReceivedHandler(S3Client s3Client,
+                                            String bucketName) {
+        this.s3Client = s3Client;
+        this.bucketName = bucketName;
+    }
+
+    @Override
+    public Void handleRequest(ScheduledEvent scheduledEvent, Context context) {
+        if (!receivedScopusEmail()) {
```

Comment:
If this is the only usage, rename the method `didNotReceiveScopusEmail` as the negation is easily ignored, otherwise create a new method.